package.nls.json

```diff
@@ -55,7 +55,7 @@
     "AWS.stepFunctions.publishWizard.stateMachineNameToUpdate.noStateMachines.detail": "Create a state machine before proceeding. See documentation for details.",
     "AWS.stepFunctions.publishWizard.stateMachineNameToUpdate.title": "Select state machine to update ({0})",
     "AWS.stepFunctions.render": "Render graph",
-    "AWS.command.samVisualize.renderTemplate": "Render CloudFormation template",
+    "AWS.command.samVisualize.renderTemplate": "Render SAM Template graph",
```

Comment:
Nit: does this mean CFN templates aren't supported?

Reply:
I'm changing that back to CloudFormation template. CFN templates are supported.